docs/root/intro/version_history.rst

```diff
@@ -19,6 +19,8 @@ Version history
 * cors: added :ref: `invalid/valid stats <cors-statistics>` to filter.
 * ext-authz: added support for providing per route config - optionally disable the filter and provide context extensions.
 * fault: removed integer percentage support.
+* grpc-json: added support for :ref:`inline descriptors
```

Comment:
inline descriptors?

Reply:
Changed it to  ignoring query parameters.